beacon_node/network/src/sync/manager.rs

```diff
@@ -555,7 +571,7 @@ impl<T: BeaconChainTypes> SyncManager<T> {
                             self.network
                                 .downvote_peer(parent_request.last_submitted_peer.clone());
                             self.request_parent(parent_request);
-                            return;
+                            break;
```

Comment:
@age please confirm `break` is equivalent to `return` here

Reply:
We want the break, so we can optionally run fork choice.. but we only want to run fork choice if we have processed a block. 
You need to check if successes > 0, which it looks like you have forgotten